idx/cassandra/cassandra.go

```diff
@@ -504,7 +504,7 @@ func (c *CasIdx) prune() {
 	for range ticker.C {
 		log.Debug("cassandra-idx: pruning items from index that have not been seen for %s", maxStale.String())
 		staleTs := time.Now().Add(maxStale * -1)
-		_, err := c.Prune(-1, staleTs)
+		_, err := c.Prune(idx.OrgIdPublic, staleTs)
```

Comment:
Doesnt look like the use of `-1` had anything to do with the PublicOrg.  So i think instead this should be 
```
_, err := c.Prune(0, staleTs)
```